### PR TITLE
updated options

### PR DESCRIPTION
**Pull request by alindner14**

### CODE REVIEW

File: Detectors/GlobalTracking/src/MatchITSTPCQC.cxx [hunk 1]
```diff
@@ -80,17 +80,35 @@ bool MatchITSTPCQC::init()
...
   mPhiTPCPhysPrim = new TH1F("mPhiTPCPhysPrim", "Phi distribution of TPC tracks (physical primary); Phi [rad]; dNdPhi", 100, 0.f, 2 * TMath::Pi());
   mFractionITSTPCmatchPhiPhysPrim = new TEfficiency("mFractionITSTPCmatchPhiPhysPrim", "Fraction of ITSTPC matched tracks vs Phi (physical primary); Phi [rad]; Eff", 100, 0.f, 2 * TMath::Pi());
   mPhiPhysPrim = new TH1F("mPhiPhysPrim", "Phi distribution of matched tracks (physical primary); Phi [rad]; dNdPhi", 100, 0.f, 2 * TMath::Pi());
   // ...till here
   mEta = new TH1F("mEta", "Eta distribution of matched tracks; Eta; dNdEta", 100, -2.f, 2.f);
-  mChi2Matching = new TH1F("mChi2Matching", "Chi2 of matching; chi2", 200, 0, 20);
-  mChi2Refit = new TH1F("mChi2Refit", "Chi2 of refit; chi2", 200, 0, 20);
+  mChi2Matching = new TH1F("mChi2Matching", "Chi2 of matching; chi2", 300, 0, 30);
+  mChi2Refit = new TH1F("mChi2Refit", "Chi2 of refit; chi2", 1000, 0, 100);
+<<<<<<< HEAD
```
Comment (chiarazampolli) [inline Detectors/GlobalTracking/src/MatchITSTPCQC.cxx]: You must have had conflicts in your PR, please, fix them.

File: Detectors/GlobalTracking/src/MatchITSTPCQC.cxx [hunk 2]
```diff
@@ -101,6 +119,13 @@ bool MatchITSTPCQC::init()
   mPhiTPCPhysPrim->Sumw2();
   mPhiPhysPrim->Sumw2();
 
+  mPtTPC->SetOption("logy");
```
Comment (chiarazampolli) [inline Detectors/GlobalTracking/src/MatchITSTPCQC.cxx:118]: This will not give you the log scale as we discussed, right?

Comment (alindner14) [reply Detectors/GlobalTracking/src/MatchITSTPCQC.cxx:118]: No, the log binning is set between L92-L107 in the updated version

File: Detectors/GlobalTracking/src/MatchITSTPCQC.cxx (hunk 1 above)
Comment (alindner14) [reply Detectors/GlobalTracking/src/MatchITSTPCQC.cxx]: I did, thank you

**Review by shahor02 — DISMISSED** [review]
Approving 1st PR to trigger the CIs

File: Detectors/GlobalTracking/src/MatchITSTPCQC.cxx [hunk 3]
```diff
@@ -80,17 +80,31 @@ bool MatchITSTPCQC::init()
...
-  mPtPhysPrim = new TH1F("mPtPhysPrim", "Pt distribution of matched tracks (physical primary); Pt [GeV/c]; dNdPt", 100, 0.f, 20.f);
   mPhiTPCPhysPrim = new TH1F("mPhiTPCPhysPrim", "Phi distribution of TPC tracks (physical primary); Phi [rad]; dNdPhi", 100, 0.f, 2 * TMath::Pi());
   mFractionITSTPCmatchPhiPhysPrim = new TEfficiency("mFractionITSTPCmatchPhiPhysPrim", "Fraction of ITSTPC matched tracks vs Phi (physical primary); Phi [rad]; Eff", 100, 0.f, 2 * TMath::Pi());
   mPhiPhysPrim = new TH1F("mPhiPhysPrim", "Phi distribution of matched tracks (physical primary); Phi [rad]; dNdPhi", 100, 0.f, 2 * TMath::Pi());
   // ...till here
   mEta = new TH1F("mEta", "Eta distribution of matched tracks; Eta; dNdEta", 100, -2.f, 2.f);
-  mChi2Matching = new TH1F("mChi2Matching", "Chi2 of matching; chi2", 200, 0, 20);
-  mChi2Refit = new TH1F("mChi2Refit", "Chi2 of refit; chi2", 200, 0, 20);
-  mTimeResVsPt = new TH2F("mTimeResVsPt", "Time resolution vs Pt; Pt [GeV/c]; time res [us]", 100, 0.f, 20.f, 100, 0.f, 2.f);
+  mChi2Matching = new TH1F("mChi2Matching", "Chi2 of matching; chi2", 300, 0, 30);
```
Comment (chiarazampolli) [inline Detectors/GlobalTracking/src/MatchITSTPCQC.cxx]: Hi, 

I think that @shahor02 mentioned a cut at 30, should we enlarge the limits here?

**Comment by chiarazampolli** [comment]
Hi @alindner14 , 

Thanks for the PR. I have a question: I remember that the axis titles were kind of cut in the QCG, could you check? If this is the case, one would need to fix it (e.g. `h->GetYaxis()->SetTitleOffset(...)`.

Can you take a look?
But I would like that this is merged for the upcoming 1M events in https://alice.its.cern.ch/jira/browse/O2-2679 - @shahor02 , can you do so if it passes the tests? If we manage, the change above and in the comments could go to a new PR, as far as I am concerned.

Chiara

**Review by shahor02 — APPROVED** [review]
Hi @alindner14 @chiarazampolli 
yes, the matching chi2 cut is 30, as in this PR, also max refit chi2 100 is fine. I've just decreased the number of bins for these 2 histos, 1000 bins for chi2 was an overkill.
Merging as the CI tests were passed (fullCI failure is unrelated).

**Comment by alindner14** [comment]
Hi @chiarazampolli, @shahor02 , I just opened a PR to change the title offset for the Y axis: https://github.com/AliceO2Group/AliceO2/pull/8108/files

**Pull request merged**